index.html

```diff
@@ -0,0 +1,62 @@
+<!-- Estructura básica del documento HTML -->
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<section>
+    <header>
+        <h1>Curriculum Vitae</h1>
+    </header>
+    <body>
+        <main class="main">
+            <div class="left-column">
+                <h1>Juan Jose Campos Valderrama</h1>
+                <h3> Informacion Personal </h3>
+                <h4> <strong>Contacto</strong></h4>
+                <p> <strong>Telefono:</strong>30553490798   <br> <br> <strong>Correo: </strong>juanjcamposv2006@gmail.com 
```

Comment:
```suggestion
                <p> <strong>Teléfono:</strong>30553490798</p>
               <p> <strong>Correo: </strong>juanjcamposv2006@gmail.com </p>
```
Lo ideal es tener una unidad por párrafo en lugar de tener todo dentro de las mismas etiquetas. Con este approach obtiene el mismo resultado y su código queda más estructurado